package.json

```diff
@@ -12,49 +12,49 @@
   },
   "dependencies": {
     "@adobe/asset-compute-commons": "^1.1.2",
-    "@adobe/cgroup-metrics": "^3.0.1",
-    "@adobe/httptransfer": "^2.0.3",
-    "@adobe/metrics-sampler": "^1.0.1",
-    "@adobe/node-fetch-retry": "^1.1.0",
+    "@adobe/cgroup-metrics": "^3.0.3",
+    "@adobe/httptransfer": "^2.1.1",
+    "@adobe/metrics-sampler": "^1.0.2",
+    "@adobe/node-fetch-retry": "^1.1.1",
     "ajv": "6.12.3",
     "clone": "^2.1.2",
     "content-type": "^1.0.4",
     "css-unit-converter": "^1.1.2",
     "data-uri-to-buffer": "^3.0.1",
     "deprecation": "^2.3.1",
-    "file-type": "^16.2.0",
-    "fs-extra": "^9.0.1",
+    "file-type": "^16.4.0",
+    "fs-extra": "^10.0.0",
     "gm": "^1.23.1",
-    "image-size": "^0.9.3",
+    "image-size": "^1.0.0",
     "jsonwebtoken": "^8.5.1",
-    "mime-types": "^2.1.28",
+    "mime-types": "^2.1.30",
     "openwhisk": "^3.21.3",
     "strip-ansi": "^6.0.0",
-    "valid-data-url": "^3.0.1",
+    "valid-data-url": "^4.0.0",
```

Comment:
were there any breaking changes?

Reply:
No, otherwise I would have written it in the description.